src/xtgeo/grid3d/_roff_grid.py

```diff
@@ -406,6 +405,7 @@ def from_file(filelike):
             for tag_name, tag_keys in translate_kws.items()
         }
         found["filedata"] = {"filetype": None}
```

Comment:
This PR should probably not contain any changes to this file.

Reply:
OK. I initially changed something; then reverted. Then isort/black in the editor force-changed stuff (e.g. empty lines or not). Will revert